src/modules/constants.ts

```diff
@@ -116,7 +116,7 @@ export const TOKEN_DATA = {
   [Network.Mainnet]: ETHER_TOKEN,
 };
 
-export const ALLOWED_NETWORKS = {
+export const SUPPORTED_NETWORKS = {
```

Comment:
I don't really know if this variable name change was necessary, but I'm good with it either way.

Reply:
I changed the name because in my mind, while yes,  we support all of those networks, they _might_ not be  `allowed` at the time, depending on what chain is the dapp deployed. Nothing serious, I just think semantically speaking, it makes more sense